bpf/lib/common.h

```diff
@@ -398,7 +398,10 @@ struct ct_state {
 	__u32 src_sec_id;
 };
 
-#define PROXY_DEFAULT_LIFETIME	360
+/* Lifetime of a proxy redirection entry is an entire day. All proxies should
+ * be using TCP keepalive to force some traffic over the connection
+ * periodically. */
+#define PROXY_DEFAULT_LIFETIME 86400
```

Comment:
Does this lifetime need to be that long? With https://github.com/cilium/cilium/pull/2860, we'll have a guarantee that Kafka connections don't live longer than ~1 minute after a redirect is removed. I don't understand yet the behavior of Envoy, but we can probably set a shorter lifetime.

Reply:
I'm happy to adjust this down again once we understand the behaviour of Envoy.